src/variant.rs

```diff
@@ -529,12 +527,6 @@ impl ToVariant for str {
     }
 }
 
-impl<T: ToVariant> From<T> for Variant {
-    fn from(value: T) -> Variant {
-        value.to_variant()
```

Comment:
This impl also had the problem that it took its argument by value, i.e. took ownership of it and consumed it, but then only used it by reference. This is rather inefficient...